test/functional/fixtures/opensearch_archiver/opensearch_dashboards_sample_data_flights/mappings.json

```diff
@@ -2,409 +2,95 @@
   "type": "index",
   "value": {
     "aliases": {
-      ".opensearch_dashboards": {
-      }
     },
-    "index": ".opensearch_dashboards_1",
+    "index": "opensearch_dashboards_sample_data_flights",
```

Comment:
I am unable to relate this change, was it bad renaming ?

Reply:
i regenerated the the data and accidentally dropped this code in the wrong location in a previous PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/260 so this fixes that.

Reply:
Thanks. That makes sense to me.